esa/Dockerfile

```diff
@@ -1,18 +1,21 @@
-FROM ghcr.io/osgeo/gdal:alpine-normal-3.8.2
+# Scalene produces SIGSEGV on the alpine-normal-3.8.2 image, so we use the
```

Comment:
Hi @arthurduf , Making sure that you saw these changes to the ESA Dockerfile, run.sh, and entrypoint.sh? Thanks!

Reply:
Hi @nemo794 , I have seen these changes and I am working on it because the new Docker image does not build on the CI/CD platform

Reply:
Ah, ok. Thank you for that update!

Reply:
@arthurduf, what errors are you seeing when building the Docker image?

Reply:
@chuckwondo : silly error about unsigned repo and unreadable key file.

W: http://archive.ubuntu.com/ubuntu/dists/jammy-backports/InRelease: The key(s) in the keyring /etc/apt/trusted.gpg.d/ubuntu-keyring-2018-archive.gpg are ignored as the file is not readable by user '_apt' executing apt-key.
W: GPG error: http://archive.ubuntu.com/ubuntu jammy-backports InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 871920D1991BC93C
E: The repository 'http://archive.ubuntu.com/ubuntu jammy-backports InRelease' is not signed.
E: Problem executing scripts APT::Update::Post-Invoke 'rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true'
E: Sub-process returned an error code

If I switch to an alpine based image I get an error when compiling scalene 

 gcc -shared build/temp.linux-x86_64-cpython-311/vendor/crdp/crdp.o -L/usr/lib -o build/lib.linux-x86_64-cpython-311/scalene/crdp.abi3.so
      creating build/temp.linux-x86_64-cpython-311/scalene
      make OUTDIR=build/temp.linux-x86_64-cpython-311/scalene ARCH=
      make: /bin/sh: Operation not permitted
      make: uname: Operation not permitted
      make: /bin/sh: Operation not permitted
      make: uname: Operation not permitted

...I will check with the rest of the team tomorrow but it seems that the Docker deployed on the CICD is way too old

Reply:
Interesting. I'm wondering how that is running in ci/cd since the error indicates user `_apt`.

Regarding attempting to get scalene to compile on an alpine image, I was able to do that, but as mentioned, _running_ scalene on an alpine image resulted in a `SIGSEGV` fault. Hence the need to move to the ubuntu-small image, which runs scalene without issue.

Reply:
We didn't solve the mystery, but we manage to build the image in our CI/CD platform.

Reply:
The one based on `ubuntu-small` or `alpine-normal`?

Reply:
ubuntu-small, I did not spent to much time on the alpine-normal because of the SIGSEV problem you found...and the get-dem is working (almost) fine on the DPS